docs/default-locale-timezone.adoc

```diff
@@ -0,0 +1,93 @@
+:page-title: @DefaultLocale and @DefaultTimeZone
+:page-description: JUnit Jupiter extensions to change the values returned from Locale.getDefault() and TimeZone.getDefault()
+
+The `@DefaultLocale` and `@DefaultTimeZone` annotations can be used to change the values returned from `Locale.getDefault()` and `TimeZone.getDefault()`.
+Both annotations work on the test class level and on the test method level.
+After the annotated element has been executed the initial default value is restored.
+
+== `@DefaultLocale`
+
+The default `Locale` is specified according to the constructors of https://docs.oracle.com/javase/10/docs/api/java/util/Locale.html[Locale].
```

Comment:
Since we're targeting Java 8, can you link to https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html instead?

Also, I think this is more explanatory:

    The default `Locale` is specified according to https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#constructor.summary[its constructors] - it needs either:
    * `language` or
    * `language` and `country` or
    * `language`, `country`, and `variant`